### DISCUSSION TITLE
Android min SDK version

### DISCUSSION THREAD

**Discussion by Jon-Salmon** [InvisibleWrench/FlutterMidiCommand#66] [Q&A] [answered]
What is the minimum supported Android version for this plugin? The example app has the min set to 25, but I've not been able to find any APIs in use that don't support SDK version 23.

**Comment by anzbert** [comment, marked as answer]
It's 24 now, which is Android Nougat v7.0, which is the lowest version that can compile and run the example app without crashing.
